### PR TITLE
Feature geopy removal

### PR DESCRIPTION
**Pull request by ARmolloy**
This PR removes the Geopy library import from RCAIDE and replaces it with a built-in code base that uses the necessary Geopy functions. This reduces the number of imports needed. The PR includes minor changes to the Geopy functions and classes but they stay the same for the most part.

### CODE REVIEW

File: RCAIDE/Framework/Analyses/Geodesics/Geodesics.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#  IMPORT
+# ----------------------------------------------------------------------------------------------------------------------  
+# RCAIDE imports
+from  RCAIDE.Library.Methods.Geodesics.Geodesics import Geodesic_Calculate
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  Calculate Distance between two coordinate locations
+# ----------------------------------------------------------------------------------------------------------------------  
+def Calculate_Distance(coord1, coord2):
+    distance = Geodesic_Calculate(coord1, coord2).kilometers
```
Comment (mclarke2) [inline RCAIDE/Framework/Analyses/Geodesics/Geodesics.py:16]: add a function header

File: RCAIDE/Framework/Analyses/Geodesics/Geodesics.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    return(distance)
+
+if __name__ == '__main__':
```
Comment (mclarke2) [inline RCAIDE/Framework/Analyses/Geodesics/Geodesics.py:19]: delete this

**Review by mclarke2 — COMMENTED** [review]
make changes before I approve

File: RCAIDE/Framework/Analyses/Geodesics/Geodesics.py (hunk 1 above)
Comment (ARmolloy) [reply RCAIDE/Framework/Analyses/Geodesics/Geodesics.py]: Done.

File: RCAIDE/Framework/Analyses/Geodesics/Geodesics.py (hunk 2 above)
Comment (ARmolloy) [reply RCAIDE/Framework/Analyses/Geodesics/Geodesics.py]: Done.

**Review by mclarke2 — APPROVED** [review]

**Pull request merged**